sdk/metric/view_test.go

```diff
@@ -469,3 +471,107 @@ func TestNewViewAggregationErrorLogged(t *testing.T) {
 	assert.Nil(t, got.Aggregation, "erroring aggregation used")
 	assert.Equal(t, 1, l.ErrorN())
 }
+
+func ExampleNewView() {
+	// Rename the "latency" instrument from the v0.34.0 version of the "http"
+	// instrumentation library as "request.latency".
+	v := NewView(Instrument{
+		Name: "latency",
+		Scope: instrumentation.Scope{
+			Name:    "http",
+			Version: "v0.34.0",
+		},
+	}, Stream{Name: "request.latency"})
+
+	stream, _ := v(Instrument{
+		Name:        "latency",
+		Description: "request latency",
+		Unit:        unit.Milliseconds,
+		Kind:        InstrumentKindSyncCounter,
+		Scope: instrumentation.Scope{
+			Name:      "http",
+			Version:   "v0.34.0",
+			SchemaURL: "https://opentelemetry.io/schemas/1.0.0",
+		},
+	})
+	fmt.Println("name:", stream.Name)
+	fmt.Println("description:", stream.Description)
+	fmt.Println("unit:", stream.Unit)
+	// Output:
+	// name: request.latency
+	// description: request latency
+	// unit: ms
+}
+
+func ExampleNewView_drop() {
+	// Set the drop aggregator for all instrumentation from the "db" library.
+	v := NewView(
+		Instrument{Scope: instrumentation.Scope{Name: "db"}},
+		Stream{Aggregation: aggregation.Drop{}},
+	)
+
+	stream, _ := v(Instrument{
+		Name:  "queries",
+		Kind:  InstrumentKindSyncCounter,
+		Scope: instrumentation.Scope{Name: "db", Version: "v0.4.0"},
+	})
+	fmt.Println("name:", stream.Name)
+	fmt.Printf("aggregation: %#v", stream.Aggregation)
+	// Output:
+	// name: queries
+	// aggregation: aggregation.Drop{}
+}
+
+func ExampleNewView_wildcard() {
+	// Set unit to milliseconds for any instrument with a name suffix of ".ms".
+	v := NewView(
+		Instrument{Name: "*.ms"},
+		Stream{Unit: unit.Milliseconds},
+	)
+
+	stream, _ := v(Instrument{
+		Name: "computation.time.ms",
+		Unit: unit.Dimensionless,
+	})
+	fmt.Println("name:", stream.Name)
+	fmt.Println("unit:", stream.Unit)
+	// Output:
+	// name: computation.time.ms
+	// unit: ms
+}
+
+func ExampleView() {
```

Comment:
Maybe call this a "custom" view, or something to distinguish it from the normal "NewView"-based views.  Probably good to add a description at the top about what this is demonstrating.

Reply:
This is named `ExampleView` so it is rendered with the `View` type in documentation.

![20221119_090729](https://user-images.githubusercontent.com/5543599/202862973-f03a735a-22cb-43ef-b58b-2aaed4f36797.png)

The name can be updated with a suffix, but that seems unneeded here given there is only one example test to associate with the `View` type.

I'll update with a comment at the top of the function explaining the significance of the example.

Reply:
Updated. Please take another look.

Please unresolve if you think we should still rename this.